lib/active_admin_import/model.rb

```diff
@@ -101,10 +101,12 @@ def csv_allowed_types
       [
           'text/csv',
           'text/x-csv',
+          'text/x-comma-separated-values',
           'text/comma-separated-values',
           'application/csv',
           'application/vnd.ms-excel',
-          'application/vnd.msexcel'
+          'application/vnd.msexcel',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.